example.py

```diff
@@ -17,8 +17,21 @@ def second_method(self):
     return self.title
 
 
-def test_example():
-    chrome = Chromeless()
+def test_example_with_session(**session_kw):
+    """
+    example usage:
+        test_example_with_session(aws_access_key_id='<YOUR ACCESS KEY ID>',
+         aws_secret_access_key='<YOUR SECRET KEY>',
+         region_name='<REGION NAME>')
+    """
+    from boto3.session import Session
+    session = Session(**session_kw)
+    chrome = Chromeless(boto_session=session)
+    test_example(chrome)
+
```

Comment:
I'd move them into pytest/tests.py

Reply:
Do I need to update/remove anything in this code for this?

Reply:
no, no problem